bindings/java/src/org/sleuthkit/datamodel/Bundle.properties

```diff
@@ -55,6 +55,12 @@ BlackboardArtifact.tskUserContentSuspected.text=User Content Suspected
 BlackboardArtifact.tskMetadata.text=Metadata
 BlackboardArtifact.tskTrack.text=GPS Track
 BlackboardArtifact.tskWebAccountType.text=Web Account Type
+BlackboardArtifact.tskScreenShots.text=Screen Shots
+BlackboardArtifact.tskDhcpInfo.text=DHCP Information
+BlackboardArtifact.tskNotifications.text=Notification
```

Comment:
Application => Applications ?

Reply:
Not sure what you mean here.

Reply:
Sorry, that was a copy and paste error. I am wondering if "Notification" should be "Notifications."

This is admittedly kind of awkward in a way that affects all of the artifact types. "Notification" makes sense as a type display name, but the primary use of these display names is in the main tree view under "Extracted Content," where a grouping tree node named "Notifications" makes more sense to me than one named "Notification." So I guess, yes, Notification => Notifications.

Reply:
Fixed